pulsar-client/src/main/java/org/apache/pulsar/client/impl/conf/ConfigurationDataUtils.java

```diff
@@ -51,20 +51,18 @@ public static ObjectMapper getThreadLocal() {
         return mapper.get();
     }
 
-    private ConfigurationDataUtils() {}
+    private ConfigurationDataUtils() {
+    }
 
     public static <T> T loadData(Map<String, Object> config,
                                  T existingData,
                                  Class<T> dataCls) {
```

Comment:
The `dataCls` becomes useless after the new change, we should remove it.

Reply:
Thanks for review !
You're right, I've done the update.